### DISCUSSION TITLE
GitHub projects Beta - why aren't milestones displayed on the tickets in the board when they were there on the old projects view?

### DISCUSSION THREAD

**Discussion by DrLoebl** [community/community#52784] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

We use multiple repositories on our boards and put the same milestones across all the different repos to aid sprint planning.  In the Classic Project Board, the milestone was displayed on the GitHub Issue ticket, which made it easy to filter on the tickets in the current sprint visually.  

![image](https://user-images.githubusercontent.com/42491601/231963692-fc3efb69-a64f-4c65-b9b0-fea187dfc242.png)

Recently this Classic Project Board became extremely sluggish and unusable and so we've switched to the new Project View.  Unfortunately this has a few issues, with a load of wasted space at the top, no full screen option, but most importantly to me, the milestones are no longer displayed on the GitHub issues:

![image](https://user-images.githubusercontent.com/42491601/231964192-57503491-c2a2-4342-a539-cd9cdd52e27f.png)

I know I can filter at the top, but that's a lot less useful when you quickly want to find any old tickets and move them across.  I guess we need to use other views that show the milestones...  Just seems a shame to have taken away some functionality for no reason...  Unless I'm missing a setting to make these visible?

**Comment by DrLoebl** [comment]
Also the completion of tasks is not displayed anymore either.

**Comment by budak7273** [comment]
It looks like Milestones can now be shown in the Visible/Hidden Fields dropdown for configuring the currently selected View
